corehq/apps/users/migrations/0028_remove_domainpermissionsmirror.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations
+
+from corehq.apps.accounting.models import BillingAccount
+from corehq.apps.enterprise.models import EnterprisePermissions
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def migrate_mirrors(apps, schema_editor):
+    DomainPermissionsMirror = apps.get_model('users', 'DomainPermissionsMirror')
+    sources = {o.source for o in DomainPermissionsMirror.objects.all()}
+    for source in sources:
+        account = BillingAccount.get_account_by_domain(source)
+        account_domains = set(account.get_domains())
+        mirror_domains = {o.mirror for o in DomainPermissionsMirror.objects.filter(source=source)}
+        EnterprisePermissions(
+            account=account,
+            is_enabled=True,
+            source_domain=source,
+            domains=list(account_domains - mirror_domains - {source}),
```

Comment:
Is this correct? I would have expected it to be either just `mirror_domains - {source}` or else `(account_domains & mirror_domains) - {source}`.

As it stands this will be all domains for the account excluding the source and any mirror domains.

Reply:
Yes, you are totally right - fa0bf6ed3f2